tbx/project_styleguide/templates/patterns/molecules/streamfield/stream_block_division.html

```diff
@@ -0,0 +1,7 @@
+{% load wagtailcore_tags %}
+
+{% if value %}
+    {% for block in value %}
+        {% include_block block with unique_id=block.id %}
+    {% endfor %}
+{% endif %}
```

Comment:
question: should we not just use `templates/patterns/molecules/streamfield/stream_block.html` for all Streamfields?

I see a number of templates with the same content. It may well be it is a project convention

Reply:
I think I'll drop `stream_block_division.html` and let the page YAML access the story container directly like I did in #335 because this pattern of having a stream_block_division.html just to link division_page.html to division_story_container.html doesn't really make sense to me.